modules/creating-custom-live-rhcos-iso.adoc

```diff
@@ -69,11 +69,11 @@ The modified `sshd_config` file overrides the default `sshd_config` file.
 $ butane -p embedded.yaml -d files-dir/ > embedded.ign
 ----
 
-. Once the Ignition file is created, you can include the configuration in a new live {op-system} ISO, which is named `rhcos-sshd-4.11.0-x86_64-live.x86_64.iso`, with the `coreos-installer` utility:
+. Once the Ignition file is created, you can include the configuration in a new live {op-system} ISO, which is named `rhcos-sshd-4.12.0-x86_64-live.x86_64.iso`, with the `coreos-installer` utility:
```

Comment:
Unsure about this file - probably needs QE

Reply:
@bergerhoffer I think this file is one the user creates, but it's based off the `rhcos-4.12.0-x86_64-live.iso` which is correct from the mirror page

Reply:
@jldohmann So you're saying that you think that these changes are okay? Do you happen to know of a QE we could request to check?

Reply:
@bergerhoffer yes, but @mike-nguyen can confirm 🙂

Reply:
The name is arbitrary.  However this section needs to be updated due to changes in butane support.  I can add comments here if you'd like or another PR can be opened.

Reply:
@mike-nguyen thanks Mike, I created a [JIRA ticket](https://issues.redhat.com/browse/OSDOCS-4872) for this. Could you add your comments there? I'll open a separate PR for these changes